regolith/helpers/l_projectahelper.py

```diff
@@ -140,32 +152,31 @@ def sout(self):
                     continue
             if rc.grant and rc.grant not in projectum.get('grants'):
                 continue
-            if rc.ended and not projectum.get('end_date'):
-                continue
             if rc.ended:
-                end_date = projectum.get('end_date')
-                if isinstance(end_date, str):
-                    end_date = date_parser.parse(end_date).date()
-                low_range = desired_date - dt.timedelta(days=num_of_days)
-                high_range = desired_date + dt.timedelta(days=num_of_days)
-                if low_range <= end_date <= high_range:
-                    end_projecta.append(projectum)
+                if projectum.get('status') not in ACTIVE_STATI:
+                    if desired_date == dt.date.today() and num_of_days == 0:
+                        projecta.append(projectum)
+                    else:
+                        if projectum.get('end_date') != None:
+                            end_date = projectum.get('end_date')
+                            if isinstance(end_date, str):
+                                end_date = date_parser.parse(
+                                    end_date).date()
+                            low_range = desired_date - \
+                                dt.timedelta(days=num_of_days)
+                            high_range = desired_date + \
+                                dt.timedelta(days=num_of_days)
+                            if num_of_days != 0:
+                                if low_range <= end_date <= high_range:
+                                    end_projecta.append(projectum)
+                            else:
```

Comment:
is this needed?  Ithink it should work with just the "<=" above, no?

Reply:
wait sorry, what do you mean by this?

Actually, I was guessing the behaviour, so I was thinking if it's set to 0 days, then it'd show all the prums finished before the set date.

Reply:
should the behaviour be that it shows finished prums for the n days before?

Reply:
yes, that's right.  prums finished in the n days preceding date.  By default date is today and n is 7 but these can be set to anything you want'

Reply:
yep, I adjusted it so it does default today + 7days back